docs/modules/ROOT/pages/index.adoc

```diff
@@ -105,33 +228,31 @@ An example endorsement target is shown below:
+. Check that the `action` field is set to `c2pa.published`, `c2pa.transcoded`, or `c2pa.repackaged`. If any other action is present, fail the *endorsement assertion* validation process for this _<<C2PA Manifest>>_ and do not proceed with further validation.
+. Check the `ingredient` field that is a member of the `parameters` object for the presence of a JUMBF URI. If the JUMBF URI is not present, or cannot be resolved to the related ingredient assertion, fail the *endorsement assertion* validation process for this _<<C2PA Manifest>>_ and do not proceed with further validation.
+. Follow the JUMBF URI link in the `ingredient` field to the ingredient assertion. Check that the URI link resolves to the single ingredient assertion in the active _<<C2PA Manifest>>._ If it does not, fail the *endorsement assertion* validation process for this _<<C2PA Manifest>>_ and do not proceed with further validation. Check that the `relationship` field is `parentOf`. If it is not, fail the *endorsement assertion* validation process for this _<<C2PA Manifest>>_ and do not proceed with further validation.
+. Check the `c2pa_manifest` field in the ingredient assertion for the presence of a hashed URI. If the hashed URI is not present, or cannot be resolved to a _<<C2PA Manifest>>,_ fail the *endorsement assertion* validation process for this _<<C2PA Manifest>>_ and do not proceed with further validation.
+. Follow the JUMBF URI link in the `c2pa_manifest` field to the ingredient’s _<<C2PA Manifest>>._ Ensure that this _ingredient_ _<<C2PA Manifest>>_ has been validated according to link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_recursively_validating_integrity_of_ingredients++[Section 15.8, “Recursively validating integrity of ingredients,” of the C2PA 2.0 Technical Specification], but do not skip the establishing of signer credential trust. If it does not validate, fail the *endorsement assertion* validation process for this _<<C2PA Manifest>>_ and do not proceed with further validation.
+. Validate the endorsement as described in link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_recursively_validating_integrity_of_ingredients++[Section 15.4, “Validate the signature,” of the C2PA Technical Specification], using the credential of the ingredient _<<C2PA Manifest>>_ signer. If a signing credential is present in the endorsement’s signature’s headers, it shall not be used for validation. If validation fails, fail the *endorsement assertion* validation process for this _<<C2PA Manifest>>_ and do not proceed with further validation.
```

Comment:
Should we also allow non-C2PA signatures, such as the "identity assertion" to also sign endorsements? E.g. a copyright holder wanting to continue to assert their status issues an endorsement to a CDN that is about to transcode their media. The CDN can then be marked as "endorsed" by the copyright holder.

Reply:
@scouten-adobe to break this into a new issue for a potential post-1.0 update